packages/block/CHANGELOG.md

```diff
@@ -6,6 +6,119 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+
+- `Header.bloom` (getter) (use `Header.logsBloom` instead)
+- **Important**: also check for `bloom` usage when passing in `Block` data (also use `logsBloom` instead), this might otherwise auto-fallback to the default value without noticing!
+- `toJSON()` method: `baseFee` property (use `baseFeePerGas` instead)
+- `toJSON()` method: `bloom` property (use `logsBloom` instead)
+
+
+### Reworked BlockHeader Constructor API
+
+While the `BlockHeader` library main constructor usage is discouraged in favor of the various static constructor methods (e.g. `BlockHeader.fromHeaderData()`), it will realistically still be directly used and this change is therefore mentioned here in the release notes.
+
+To align with other libraries and simplify usage the constructor has been reworked in PR [#1787](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1787) to take in a `headerData` dict as a first argument instead of a chain of single parameters (`constructor(parentHash, uncleHash, coinbase,...)`).
+
+In doubt check if you have direct constructor usage in your code and update accordingly!
+
+### Reduced Bundle Size (MB -> KB)
+
+The bundle size of the library has been dramatically reduced going down from MBs to KBs due to a reworked genesis code handling throughout the library stack in PR [#1916](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1916) allowing the `Common` library to now ship without the bundled (large) genesis state definitions (especially for mainnet).
+
+### Removed Genesis Functionality
+
+PR [#1916](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1916) reworks the genesis code throughout the EthereumJS library stack, see also the bundle size note above.
+
+In the `Block` library the `initWithGenesisHeader` option and the `Block.genesis()` static constructor have been removed, respectively parts of the functionality moved to the `Blockchain` class where genesis block creation is mostly needed (within the realm of the EthereumJS libraries).
+
+It is still possible to create genesis blocks with the library but it is now needed to explicitly pass in the respective genesis parameters.
+
+Get in touch if you strongly rely on this part of functionality, it might be possible to provide additional helpers here which fit in the current scheme of the refactored libraries structure (providing extra functionality for the `Block` library to directly take in Geth genesis JSON files e.g. or additional helpers for the `Blockchain` package).
+
+### Removed Blockchain-based Validation Methods
+
+In the former code base of the `Block` libraries various validation methods (and therefore the whole library) depended on passing in a `Blockchain` instance, since context from the broader `Blockchain` (in many cases the parent block) was needed for validation. This was an unlucky situation since it lead to a somewhat circular dependency situation (a `Block` should not depend on a `Blockchain`).
```

Comment:
```suggestion
In the former code base of the `Block` libraries various validation methods (and therefore the whole library) depended on passing in a `Blockchain` instance, since context from the broader `Blockchain` (in many cases the parent block) was needed for validation. This was an unlucky situation since it led to a somewhat circular dependency situation (a `Block` should not depend on a `Blockchain`).
```